otter/convergence/composition.py

```diff
@@ -66,3 +46,36 @@ def tenant_is_enabled(tenant_id, get_config_value):
     """
     enabled_tenant_ids = get_config_value("convergence-tenants")
     return (tenant_id in enabled_tenant_ids)
+
+
+def json_to_LBConfigs(lbs_json):
```

Comment:
I don't have strong opinion but I would argue that it is more related to gathering as this function is based on raw JSON knowledge of gathered response. It's also ok to put it in model. I am actually fine either way. Probably just bike-shedding. You can ignore it.